main.py

```diff
@@ -1,10 +1,14 @@
+import argparse
 import pickle
 from sklearn.ensemble import RandomForestClassifier
 from feature_importance import plot_importances
 
-path = './PathToFile/'
+parser = argparse.ArgumentParser()
+parser.add_argument("path", help="Input a directory path where the models are.")
+args = parser.parse_args()
+
 model_names = ['model_name1', 'model_name2', 'model_name3']
```

Comment:
If your path is an input, your model names should be too...
```
parser.add_argument('--nargs', nargs='+')
```

Then you can call like
```
python main.py --nargs 1234 2345 3456 4567
['1234', '2345', '3456', '4567']
```

Reply:
Should model names be a positional argument instead of an optional argument? I also add an optional argument for saving figure.